mocks/mock-express-usage-record.js

```diff
@@ -61,6 +61,9 @@ function genExpressMetrics(cb) {
     method: 'POST'
   }];
 
+  var dataStartTime = new Date(new Date() - 23 * 60 * 60 * 1000);
+  console.log('Data starts at %s', dataStartTime.toString());
```

Comment:
Isn't `.toString()` redundant here? Personally I'd rather see `%j` or `.toJSON()` used, but this is just for tests AFAICT...